src/components/signup/ImagePicker/ImagePicker.tsx

```diff
@@ -0,0 +1,65 @@
+import { useRef, useState } from "react";
+import { useFormContext } from "react-hook-form";
+
+export interface IImagePickerProps {
+  name: string;
+}
+
+const ImagePicker = ({ name }: IImagePickerProps) => {
+  const { register, setValue } = useFormContext();
+
+  const [pickedImage, setPickedImage] = useState<string | null>(null);
+
+  const imageInput = useRef<HTMLInputElement>(null);
+
+  const buttonClickHandler = () => {
+    imageInput.current?.click();
+  };
+
+  const handleImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+
+    if (!file) {
```

Comment:
```suggestion
if (file) {
      const fileReader = new FileReader();

      fileReader.onload = () => {
        if (typeof fileReader.result === 'string') {
          setPickedImage(fileReader.result);
        }
      };

      fileReader.readAsDataURL(file);

      setValue(name, file);
}
```
파일이 있는 경우로 if문을 바꾸면 파일이 없는 경우 코드는 없어져도 될 것 같은데, 위 방법은 어떤지 의견 부탁드립니다!

Reply:
넵 말씀하신대로 수정했습니다! 👍👍👍